### ISSUE TITLE
Doubt about variable 'execution_time'

### ISSUE THREAD

**Issue by ilska** [dfki-ric/movement_primitives#24]
Hi,

I am trying to understand your implementation and when analyzing the formula related to the rollout of the DMP, (see this line: 
https://github.com/dfki-ric/movement_primitives/blob/be115a9623e4b9f87e472f66708a9f70417fc157/movement_primitives/dmp/_dmp.py#L275), I am not sure about the relation between the variable 'execution_time' and the 'tau' variable from the theory of the paper 'Dynamical movement primitives: Learning attractor models for motor behaviors' .

The division with the squared of 'execution_time' in https://github.com/dfki-ric/movement_primitives/blob/be115a9623e4b9f87e472f66708a9f70417fc157/movement_primitives/dmp/_dmp.py#L279 confuses me. Would you have any reference paper where I could check this up?

Thank you for your time.

**Comment by AlexanderFabisch** [comment]
Hi, this is Equation 2.1 of the paper solved for the second derivative of y. Hence, tau has to be squared.

**Comment by ilska** [comment]
True! So that means that tau  is actually execution_time, isn't it? 
Thanks for your fast response.

**Comment by AlexanderFabisch** [comment]
Yes, it is
